core/performance_test/benchmark/Cabana_LinkedCellPerformance.cpp

```diff
@@ -0,0 +1,169 @@
+/****************************************************************************
+ * Copyright (c) 2018-2021 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "Cabana_BenchmarkTimer.hpp"
+
+#include <Cabana_Core.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <numeric>
+#include <sstream>
+#include <string>
+#include <vector>
+
+//---------------------------------------------------------------------------//
+// Performance test.
+template <class Device>
+void performanceTest( std::ostream& stream, const std::string& test_prefix )
+{
+    // Declare problem sizes.
+    double min_dist = 1.0;
+    std::vector<int> problem_sizes = { 1000, 10000, 100000, 1000000 };
```

Comment:
Should we consider increasing this to up to 1e7? I guess it depends on current run times. Some GPU models may use a lot of particles.

Reply:
I meant to ask whether we should reduce these values to run in the CI, or increase to run in production

Reply:
I guess we could just edit for production runs